src/authorization.ts

```diff
@@ -13,7 +13,13 @@ export interface AuthorizationRequest {
     resourceType?: string;
     id?: string;
     vid?: string;
-    exportType?: ExportType;
+    export?: ExportAuth;
+}
+
+export interface ExportAuth {

```

Comment:
perhaps rename to BulkAuth (if import will need this)

Reply:
We haven't built a tech design for import yet, but assuming import follow this format. 
```
1. Generate presigned URL for import. (Authorize that user has write permission for resources they're looking to import)
    a. User upload file to s3
    b. If user needs to upload more files, they can request more presigned S3 urls 
    c. User send request signifying they're done uploading files
2. User get status of import job. (Authorize that user is same as creator of job)
3. User can cancel import job
```
Given this import would require the `operation` field and the `jobRequesterUserId` field. However, it wouldn't need the `type?: ExportType` field.

Instead it would probably use a field called `importTypes: string[]`. This would be an array of resources that the user is looking to import. We will need to check if the user has write permission on those resources.

Reply:
What are your thoughts here? Should we include `importTypes` here and then allow this interface to serve both Export and Import? I feel like it might be more DRY but it would be more confusing, as all of the fields will never be used. `type` and `importResources` are mutually exclusive.

Reply:
hmm maybe we have different types but in the return we use the `|` So it would look like:
```typescript
 bulkAuth?: ExportAuth | ImportAuth;
```

Reply:
Given our offline discussion with using the persistence layer to validate `get-status` and `cancel-export`, we no longer need need `jobRequesterUserId` anymore. We can now just use `bulkDataAuth`